tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNet/AspNetWebApi2Integration.cs

```diff
@@ -146,10 +146,13 @@ internal static void UpdateSpan(IHttpControllerContext controllerContext, Span s
                     tags,
                     headerTags);
 
-                tags.AspNetAction = action;
-                tags.AspNetController = controller;
-                tags.AspNetArea = area;
-                tags.AspNetRoute = route;
+                if (tags is not null)
```

Comment:
As we add null checks here and in DecorateWebServerSpan I'd say it would make sense to make the tag argument nullable.